libraries/AC_AutoTune/AC_AutoTune_Multi.cpp

```diff
@@ -440,6 +440,30 @@ void AC_AutoTune_Multi::save_tuning_gains()
     reset();
 }
 
+// send final gains for a given axis to GCS
+void AC_AutoTune_Multi::send_final_gains(AxisType test_axis) const
+{
+    switch (test_axis) {
+        case ROLL:
+            send_axis_gains("Roll", tune_roll_rp, tune_roll_rp*AUTOTUNE_PI_RATIO_FINAL, tune_roll_rd, tune_roll_sp, tune_roll_accel);
+            break;
+        case PITCH:
+            send_axis_gains("Pitch", tune_pitch_rp, tune_pitch_rp*AUTOTUNE_PI_RATIO_FINAL, tune_pitch_rd, tune_pitch_sp, tune_pitch_accel);
+            break;
+        case YAW:
+            send_axis_gains("Yaw", tune_yaw_rp, tune_yaw_rp*AUTOTUNE_YAW_PI_RATIO_FINAL, 0, tune_yaw_sp, tune_yaw_accel);
+            break;
+    }
+}
+
+// send gain formating helper
+void AC_AutoTune_Multi::send_axis_gains(const char* axis_string, float rate_P, float rate_I, float rate_D, float angle_P, float max_rate) const
+{
+    gcs().send_text(MAV_SEVERITY_NOTICE,"AutoTune: %s complete", axis_string);
```

Comment:
Again not a blocker but at some point we could reduce flash by replacing all the occurances of "AutoTune:" with a const string.